src/style.css

```diff
@@ -0,0 +1,7 @@
+@import "./card.css";
+@import "./stars.css";
+
+* {
+  margin: 0;
```

Comment:
why do we need to reset margins for `*` selector
reset only where needed

Reply:
didn't change it